composer.json

```diff
@@ -139,7 +139,7 @@
         "spryker/glue-storefront-api-application-authorization-connector": "^1.1.0",
         "spryker/glue-storefront-api-application-glue-json-api-convention-connector": "^1.0.0",
         "spryker/message-broker": "^1.9.0",
-        "spryker/message-broker-aws": "^1.6.0",
+        "spryker/message-broker-aws": "1.7.0",
         "spryker/money-gui": "^1.1.0",
```

Comment:
Why we do not have `ReservationVersionPostSaveTerminationAwareStrategyPlugin` related change as in other PRs?

Reply:
because it is already present